### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by rimmesbe** [tcarisch/Capstone-project#6]
Grade: needs improvement

Summary: You have a good start to the project. Obviously there is a lot of work left to do. Take a look at the issues I created for some tips on improvement and best practices. If you have the time I would work more on the project to really get it looking good. Hope you enjoyed the course!

**Comment by tcarisch** [comment]
Yeah, I realized that so much more could be improved upon.  Just wanted to make the deadline.
